aws-sso.rb

```diff
@@ -49,13 +49,16 @@ def install
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
 
-  # TODO: re-add tests
-
   def caveats; <<-EOS.undent
       aws-sso requires some setup before it's ready to use. Run `aws-sso setup` to set up a config file.
 
       For more detailed info, see the README on Bitbucket: https://bitbucket.org/ensighten-ondemand/aws_sso/src/master/README.md
     EOS
   end
-end
 
+  test do
+    system "true"
```

Comment:
The program currently has no options that don't require input. They all also have side-effects. @thejandroman has added a wishlist feature to add --version checks, so when I add those we can use that.